tests/installation/select_patterns_and_packages.pm

```diff
@@ -190,8 +191,8 @@ sub run {
                 };
                 assert_screen 'current-pattern-selected', 5;
             }
-            $self->workaround_dependency_issues;
-            # stick to the default patterns
+            $dep_issue = $self->workaround_dependency_issues || $dep_issue;    # check if at least 1 dep. issue was displayed
+                                                                               # stick to the default patterns
```

Comment:
better to split the comments from the code, i.e. move the comment in the previous above the code and not next to it

Reply:
tidy put together, yes, better on the top.